origin-js/src/resources/marketplace.js

```diff
@@ -643,15 +659,24 @@ export default class Marketplace {
             })
           )
         } catch(e) {
+          // TBD: what exactly are we guarding against here ?
```

Comment:
@franckc see [this](https://github.com/OriginProtocol/origin-js/issues/558) and probably [this](https://github.com/OriginProtocol/origin/issues/563) can be closed.

Reply:
Thanks for pointing this out. Makes sense - we want to protect against invalid offers.
Will add comment. 

Will also add checks in other places of the notification code to harden it further: when fetching listing in this method, and also at the adapter level in the getNotification method (for ex an invalid IPFS Hash could cause that layer to error out).